lib/livebook/hubs/team.ex

```diff
@@ -38,7 +38,7 @@ defmodule Livebook.Hubs.Team do
     field :org_key_id, :integer
     field :teams_key, :string
     field :org_public_key, :string
-    field :session_token, :string
+    field :session_token, :string, redact: true
```

Comment:
Should we do that for the teams key too?

Reply:
The teams key is not strictly necessary and I thought having it in logs may help if someone loses it :P